tuf/src/metadata.rs

```diff
@@ -3036,7 +3036,7 @@ mod test {
     //                              (,,(,    \.         `.   ____,-`.,
     //                           (,'     `/   \.   ,--.___`.'
     //                       ,  ,'  ,--.  `,   \.;'         `
-    //                        `{o, {    \  :    \;
+    //                        `(o, /    \  :    \;
```

Comment:
did not expect clippy to have preferences on the depictions of wyrms

Reply:
This wasn't *really* a clippy lint, but those `{` always screwed up bracket matching in vim and vscode, so I decided to finally get rid of it.